### ISSUE TITLE
Update copy for Lottery Factor

### ISSUE THREAD

**Issue by BekahHW** [open-sauced/app#3680]
Based on feedback, there's concern over identifying the "risk" to a project. It's important that we use language that allows the users to see that it's a piece of the overall picture of the repository health.  

## Current Copy

<img width="336" alt="image" src="https://github.com/open-sauced/app/assets/34313413/18b192ac-cefe-426c-a56a-a6c740ec5b1c">

## Proposed Copy for Now

Identifies project dependency on top contributors.

**Comment by BekahHW** [comment]
> We should also be emphasizing that we have comments in addition to commits. cc @isabensusan

@bdougie are you referring to issue comments on the user profile?

**Comment by bdougie** [comment]
> @bdougie are you referring to issue comments on the user profile?

Mostly sharing there is an opportunity that we can add a new devstat show folks who comment the most. In the origin OSCR doc I made internally, I called this Sway. 

_Sway: noun - The control or influencing conversation or situation._  

To the point of a maintainer, there are more things than just code happening in a project, but we are only showing that on the repo page. Reviews and comments can be represented with Sway.
